### ISSUE TITLE
Batching only half working (ubuntu, Bob 2.0)

### ISSUE THREAD

**Issue by kitschbent** [kyokan/bob-wallet#613]
My non-technical contribution...

The batching function in 2.0 isn't working fully. (Rithvik you might recall you linked me to a preview of this some months ago to fix an issue of mine, so the comparison is between the modified 1.0 I was using (yours) and batching in 2.0). My assumption is that 2.0 batching should be more optimized, a la the modified 1.0. 

Let's say I have....  739 names I'd like to transfer from one wallet to another.

These should ideally be batched into the following Txs

100,100,100,100,100,100,100,39

In the modified 1.0, what would happen is these would be batched in 8 Txs, as represented above. And it worked terrifically. 

In 2.0 batching (I'm on Ubuntu), *only* the first 100 are sent. The remaining 7 Txs never appear in the debug log at all, I'm only finding the creation of 1 single Tx. And, it works fine, goes to mempool, and  finalizes just like it should. 

So, batching is sort of halfway working.

**Comment by rithvikvibhu** [comment]
You're right, in https://github.com/kyokan/bob-wallet/pull/546 the behavior was to send all N txs at once, but with https://github.com/kyokan/bob-wallet/pull/569, we changed it to one tx at a time, this was intentional:
- One is because Ledger users would find it painful (imagine verifying 500 names, each of which requires 4-5 clicks!) There's no way to skip the confirm, or to pick a custom number of names for the bulk actions right now. And if it fails in the middle (rare, but happens) then have to start all over again.
- And with multisig, the wallet code was refactored to to be more flexible (hot/cold, single/multisig, etc.) This changed a lot of things and it made sense to keep things as simple as possible.

I think we can add that back in once:
- It's possible to view and select what goes in the batch i.e. being able to select a smaller set of actions
- Ledger users don't have to wear out their fingers (need more discussion on how)
- Multisig UI with multiple txs is figured out
